sdk/tvl-usd.js

```diff
@@ -0,0 +1,138 @@
+const debug = require("debug")("opentvl:tvl-usd");
```

Comment:
the file itself seems to better be part of server instead of being part of the sdk. a few reasons:

1. it is stateful. feeds need to be prefetched (or maybe not for the initial version for simplicity, with the cost of speed?), some more concerns described in line 14
2. a specific adapter don't have to worry about mapping all tokens into USD - this is a general issue that server should handle for all adapters. For a specific adapter, they can use `sdk.eth.chainlink.getPrices([{token0, token1}, ...])` to get price for many pairs if they would like to.